Changelog.md

```diff
@@ -7,11 +7,22 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
 ## Next Release (Date TBD)
 
 #### New experimental features
-- [#615](https://github.com/timescale/timescaledb-toolkit/pull/614): Heatbeat aggregate
+- [#615](https://github.com/timescale/timescaledb-toolkit/pull/615): Heartbeat aggregate
```

Comment:
fixed a couple typos here and added blanks to unsquash everything.  Without the blanks, github renders as run-on sentence "Heartbeat aggregate Users can use..."